mtags-interfaces/src/main/java/scala/meta/pc/CompilerFiles.java

```diff
@@ -0,0 +1,6 @@
+package scala.meta.pc;
+
+public interface CompilerFiles {
```

Comment:
Let's add some docs why it's needed.

The alternative is to not add new method, but send changedFiles for every file if the project doesn't compile

Reply:
I moved the files to outline logic outside of pc. I think it increases readability of that logic.